### DISCUSSION TITLE
Higher build plate temperature?

### DISCUSSION THREAD

**Discussion by corrosion** [aegean-odyssey/mpmd_marlin_1.1.x#68] [Q&A]
Does this firmware allow increasing the bed temp above 60C? Assuming a proper external power supply is used..

**Comment by aegean-odyssey** [comment]
The `mpmd_marlin_1.1.x` firmware sets the maximum bed temperature at 150&deg;C. With a proper power supply (as you mention), the major limiting factors are the power handling limits of the bed heating element and the switching transistor used to control power to the heating element. I don't know about the heating element, but the switching transistor (MOSFET), _I think_, has a maximum current rating of 8 Amps. 

A side note: while trying out the PID tuning commands,  the printer here has reached a bed temperature of 70&deg;C with the supplied (60W) power supply.
